tests/unit/cloudformation/test_connection.py

```diff
@@ -149,6 +150,11 @@ def test_update_stack_all_args(self):
             'NotificationARNs.member.2': 'arn:notify2',
             'Parameters.member.1.ParameterKey': 'KeyName',
             'Parameters.member.1.ParameterValue': 'myKeyName',
+            'Parameters.member.2.ParameterKey': 'KeyName2',
+            'Parameters.member.2.ParameterValue': '',
```

Comment:
You may need to update the logic to exclude ``ParameterValue`` if ``UsePreviousValue`` is being used. It looks like it works according to the integration test, but I would like to stick with what the API references say, which is to exclude a parameter value if specify UsePreviousValue is true: http://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_Parameter.html